dash/App/Chains/Chain/ChainExpanded/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react'
+import link from '../../../../../resources/link'
+
+import Connection from '../Connection'
+
+import { SubmitChainButton, ChainHeader, EditChainColor, EditChainName, EditChainSymbol, EditChainId, EditTestnet, EditChainExplorer, ChainFooter } from '../Components'
+
+const updateChain = (chain, updatedChain) => {
+  link.send('tray:action', 'updateNetwork', chain, updatedChain)
+}
+
+export default (props) => {
+  // props
+  const { id, name, type, explorer, symbol, isTestnet, filter, on, connection, primaryColor, icon, price } = props
+  const chain = { id, type, name, isTestnet, symbol, explorer, primaryColor }
+
+  // state
+  const [currentColor, setPrimaryColor] = useState(primaryColor)
+  const [currentName, setName] = useState(name)
+  const [currentSymbol, setSymbol] = useState(symbol)
+  const [currentExplorer, setExplorer] = useState(explorer)
+  const [currentTestnet, setTestnet] = useState(isTestnet)
+
+  const updatedChain= {
+    id,
+    type,
+    name: currentName,
+    primaryColor: currentColor,
+    isTestnet: currentTestnet,
+    symbol: currentSymbol,
+    explorer: currentExplorer
+  }
+
+  return (
+    <div key={'expandedChain'} className='network cardShow'>
+      <ChainHeader 
+        type={type}
+        id={id}
+        primaryColor={currentColor}
+        icon={icon}
+        name={currentName}
+        on={on}
+        showToggle={true}
+      />
+      <EditChainColor 
+        currentColor={primaryColor} 
+        onChange={color => {
+          setPrimaryColor(color)
+          link.send('tray:action', 'setChainColor', chain.id, color)
+        }}
+      />
+      <EditChainName
+        currentName={currentName}
+        onChange={name => {
+          setName(name)
+          updatedChain.name = name
+          updateChain(chain, updatedChain)
+        }}
+      />
+      <EditChainExplorer
+        currentExplorer={currentExplorer}
+        onChange={explorer => {
+          setExplorer(explorer)
+          updatedChain.explorer = explorer
+          updateChain(chain, updatedChain)
+        }}
+      />
+      <EditChainSymbol
+        currentSymbol={currentSymbol}
+        onChange={symbol => {
+          setSymbol(symbol)
+          updatedChain.symbol = symbol
+          updateChain(chain, updatedChain)
+        }}
+      />
+      <ChainFooter 
+        symbol={symbol} 
+        price={price} 
+      />
+      {id !== 1 ? (
+        <EditTestnet
+          testnet={currentTestnet}
+          onChange={testnet => {
+            setTestnet(testnet)
+            updatedChain.isTestnet = testnet
+            updateChain(chain, updatedChain)
+          }}
+        />
+      ) : null}
+      <div className='chainModules'>
+        <Connection expanded={true} connection={connection} {...chain} />
```

Comment:
do we need to allow the ability to edit these in 2 places? feels like it would be fine to have them in one place or the other, but not both

Reply:
So the plan is for the expanded chain to be a fully featured chain management interface that has all the data and config functionality in one place. I didn't want to split up chain configuration tasks between two views. The preview version (in the future) is meant to only be the things you want to see quickly -- for example in the future you can build out your RPC list in the expanded view but see the currently connected RPC in the preview. Until I have time to update that Connection module this seem'd like an ok place to land as it is open by default in the expanded view and closed by default in the preview.